### PR TITLE
Covered all the ACS section I and II topics adequately

### PR DESCRIPTION
**Pull request by petercmonaco**

### CODE REVIEW

File: Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md [hunk 1]
```diff
@@ -33,19 +33,48 @@ Ground 1.5-3 hours, depending on memory of concepts from [[PPL]]
...
 	3. **Regulations**: Introduction of commercial operations requires a wider and deeper understanding of the [[FAR]]s
 	4. **Maneuvers**: [[PPL]] maneuvers are mostly 1 dimensional, [[CPL]] requires maneuvering simultaneously in 2 dimensions
-3. Preflight Preparation
-	1. May carry persons or property for hire subject to common carriage vs private carriage limitations
-		1. [[Private and Commercial Pilot Privileges and Limitation]] #todo :: Add source, maybe [[AC 61-142]]
-		2. [[Distinction Between Pilot Privileges and Operational Rules]] #todo :: Add source
-	2. [[CPL]]+[[IRA]] allows commercial flights >50nm and at night.
+3. [[Commercial ACS]] I. Preflight Preparation
+	1. [[Commercial ACS]] I.A. Pilot Qualifications
+		1. Builds on the foundation in [[Complete VFR Preflight Checklist]]/Pilot section
```
Comment (johnjackoleary) [inline Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md:38]: Can do [[Complete VFR Preflight Checklist#Pilot to link direct]]

File: Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md [hunk 2]
```diff
@@ -33,19 +33,48 @@ Ground 1.5-3 hours, depending on memory of concepts from [[PPL]]
...
+		2. Review [[FAR 61.133]]
+		3. May carry persons or property for hire subject to common carriage vs private carriage limitations
+		4. [[Private and Commercial Pilot Privileges and Limitation]] #todo :: Add source, maybe [[AC 61-142]]
+		5. [[Distinction Between Pilot Privileges and Operational Rules]] #todo :: Add source
+		6. [[CPL]]+[[IRA]] allows commercial flights >50nm and at night.
+		7. Medical Certificate requirements
+	2. [[Commercial ACS]] I.B. Airworthiness Requirements
+		1. Review [[Complete VFR Preflight Checklist]]/Aircraft section
```
Comment (johnjackoleary) [inline Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md:46]: See comment above

File: Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md [hunk 3]
```diff
@@ -33,19 +33,48 @@ Ground 1.5-3 hours, depending on memory of concepts from [[PPL]]
...
+			6. [[XC.6 Pilotage and Dead Reckoning]]
+			7. [[XC.7 Making the Go-No-Go Decision]]
+	5. [[Commercial ACS]] I.E. National Airspace System
+		1. Review [[SOLO.7 Navigation Charts and National Airspace System]]
+	6. [[Commercial ACS]] I.F. Performance and Limitation
+		1. Review [[XC.4 Aircraft Performance and Weight+Balance]]
+	7. [[Commercial ACS]] I.G. Operation of Systems
+		1. Review.... ?????
+	8. [[Commercial ACS]] I.H. Human Factors
+		1. Review... ?????
```
Comment (johnjackoleary) [inline Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md:68]: Currently, a file called [[Human Factors]], which is in XC.5. [[Human Factors]] could definitely use some love though

File: Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md [hunk 4]
```diff
@@ -33,19 +33,48 @@ Ground 1.5-3 hours, depending on memory of concepts from [[PPL]]
...
+			4. [[XC.4 Aircraft Performance and Weight+Balance]]
+			5. [[XC.5 Advanced Weather Briefs and Preflight Planning]]
+			6. [[XC.6 Pilotage and Dead Reckoning]]
+			7. [[XC.7 Making the Go-No-Go Decision]]
+	5. [[Commercial ACS]] I.E. National Airspace System
+		1. Review [[SOLO.7 Navigation Charts and National Airspace System]]
+	6. [[Commercial ACS]] I.F. Performance and Limitation
+		1. Review [[XC.4 Aircraft Performance and Weight+Balance]]
+	7. [[Commercial ACS]] I.G. Operation of Systems
+		1. Review.... ?????
```
Comment (johnjackoleary) [inline Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md:66]: Assumed to have been learned in ground school, but perhaps it's better we add a specific point in the lesson plans where this is confirmed to be learned (or taught if not yet learned). Would want it to be before XC.9 (or maybe even part of that lesson, and make it 3 hours long..?)

File: Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md [hunk 5]
```diff
@@ -33,19 +33,48 @@ Ground 1.5-3 hours, depending on memory of concepts from [[PPL]]
...
-	2. [[CPL]]+[[IRA]] allows commercial flights >50nm and at night.
+3. [[Commercial ACS]] I. Preflight Preparation
+	1. [[Commercial ACS]] I.A. Pilot Qualifications
+		1. Builds on the foundation in [[Complete VFR Preflight Checklist]]/Pilot section
+		2. Review [[FAR 61.133]]
+		3. May carry persons or property for hire subject to common carriage vs private carriage limitations
+		4. [[Private and Commercial Pilot Privileges and Limitation]] #todo :: Add source, maybe [[AC 61-142]]
+		5. [[Distinction Between Pilot Privileges and Operational Rules]] #todo :: Add source
+		6. [[CPL]]+[[IRA]] allows commercial flights >50nm and at night.
+		7. Medical Certificate requirements
```
Comment (johnjackoleary) [inline Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md:44]: Could link to [[Medical Requirements]]

**Review by johnjackoleary — COMMENTED** [review]
Few minor things, generally I like it and looks good to me

File: Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md (hunk 1 above)
Comment (petercmonaco) [reply Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md:38]: Thanks!

File: Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md (hunk 4 above)
Comment (petercmonaco) [reply Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md:66]: I'm going to punt on this for now, and just reference the PHAK chapter.

File: Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md (hunk 5 above)
Comment (petercmonaco) [reply Lesson Plans/Commercial/CPL.1 Commercial Overview, Preflight Preparation and Procedures, and Airport Operations.md]: Done.  And I improved that page.

**Pull request merged**